update_prevalence.py

```diff
@@ -18,6 +19,25 @@
 from typing import Optional, ClassVar, Iterator, List, Dict, Type, TypeVar, Any
 from us_state_abbrev import us_state_name_by_abbrev
 
+VACCINE_MULTIPLIERS = {
```

Comment:
Is there a way to reference the existing vaccine multiplier constants, instead of duplicating them here? I'm sure this list will change at some point - best to have a single copy so we don't have to track them all down for updating.

Reply:
Yeah! I can read them from the csv file.

We still have 2 copies, but at least it's not more.